packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/OpenstackDetailsSection.tsx

```diff
@@ -39,9 +40,13 @@ export const OpenstackDetailsSection: React.FC<DetailsSectionProps> = ({ data })
       <URLDetailsItem
         resource={provider}
         canPatch={permissions.canPatch}
-        helpContent={t(
-          `OpenStack Identity (Keystone) endpoint, for example, http://controller:5000/v3`,
-        )}
+        helpContent={
+          <Trans t={t} ns="plugin__forklift-console-plugin">
+            {
+              'URL of the OpenStack Identity (Keystone) endpoint. For example: <strong>https://identity_service.com:5000/v3</strong>.'
+            }
+          </Trans>
```

Comment:
```suggestion
          <Trans t={t} ns="plugin__forklift-console-plugin">
              [ reactnode ]
          </Trans>
```

can you replace all the places you send a string to `Trans` ?

Reply:
There are few places to replace for other providers. I'll fix it in a follow up PR, together with doc review fixes.